app/utilities/first_time_data_generator.rb

```diff
@@ -1082,15 +1616,15 @@ def generate_module_comment(my_module, user, message, created_at = nil)
       my_module: my_module,
       created_at: created_at,
       updated_at: created_at,
-      message: t('activities.add_comment_to_module',
+      message: I18n.t('activities.add_comment_to_module',
                  user: user.full_name,
                  module: my_module.name)
     ).sneaky_save
   end
 
   def generate_result_comment(result, user, message, created_at = nil)
     created_at ||= generate_random_time(result.created_at, 1.days)
-    ResultComment.new(
+    ResultComment.create(
```

Comment:
Should be left `new`, since `sneaky_save` is used afterwards?

Reply:
`sneaky_save` is not working. I don't know why. None of `ResultComment` ever gets created. Do we need to use `sneaky_save`, or we can change it with `create`?